tests/integration/golang/fixtures/metric.go

```diff
@@ -74,13 +76,17 @@ func (f MetricFixtures) GetMetricsByContext(
 func (f MetricFixtures) CreateLatestMetric(
 	ctx context.Context, metric *models.LatestMetric,
 ) (*models.LatestMetric, error) {
-	if metric.Context != nil {
+	if err := f.baseFixtures.db.WithContext(ctx).FirstOrCreate(&models.DefaultContext).Error; err != nil {
+		return nil, eris.Wrap(err, "error creating or finding default context")
+	}
+	if metric.Context.Json == nil {
+		metric.Context = models.DefaultContext
+	} else {
 		if err := f.baseFixtures.db.WithContext(ctx).FirstOrCreate(
 			&metric.Context, metric.Context,
 		).Error; err != nil {
-			return nil, eris.Wrap(err, "error creating latest metric context")
+			return nil, eris.Wrap(err, "error creating metric context")
 		}
```

Comment:
just a tiny pick -> you don't need `else` statement here. will be enough to have only `if` statement as I can se.

Reply:
ahh, sorry, yes.

Reply:
moment =)

Reply:
but if you remove:
```
if err := f.baseFixtures.db.WithContext(ctx).FirstOrCreate(&models.DefaultContext).Error; err != nil {
 		return nil, eris.Wrap(err, "error creating or finding default context")
 	}
```
and stay:
```
if metric.Context.Json == nil {
 		metric.Context = models.DefaultContext
 	} 


 		if err := f.baseFixtures.db.WithContext(ctx).FirstOrCreate(
 			&metric.Context, metric.Context,
 		).Error; err != nil {
 			return nil, eris.Wrap(err, "error creating metric context")
 		}
```
it should be the same and more compact?